scripts/action-handlers/item-macro-extender.js

```diff
@@ -18,6 +18,10 @@ export class ItemMacroActionListExtender extends ActionListExtender {
      * @override
      */
     extendActionList () {
+        // Update actor and token with current action handler context
+        this.actor = this.actionHandler.actor;
```

Comment:
These variables will almost always be null at construction time, and need to be updated when called.